### PR TITLE
[#2054] Fix cstyle error in ./src/istgt_crc32c.c

### PR DESCRIPTION
**Pull request by nfayazov**
Signed off by: Nadir Fayazov nfayazov@gmail.com

**What this PR does / why we need it:**
Fix cstyle error in ./src/istgt_crc32c.c

### CODE REVIEW

File: src/istgt_crc32c.c [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 /* defined in RFC3720(12.1) */
 static uint32_t istgt_crc32c_initial    = ISTGT_CRC32C_INITIAL;
-static uint32_t istgt_crc32c_xor        = ISTGT_CRC32C_XOR;
+static uint32_t istgt_crc32c_xor			 = ISTGT_CRC32C_XOR;
```
Comment (utkarshmani1997) [inline src/istgt_crc32c.c]: @nfayazov fix indentation here

**Review by utkarshmani1997 — CHANGES_REQUESTED** [review]
@nfayazov changes look good to me. Have given comment to fix indentation.

**Comment by agarwal-akash** [comment]
@utkarshmani1997 PR to this issue has already been submitted yesterday [#2054](https://github.com/openebs/istgt/pull/123)

**Comment by nfayazov** [comment]
The other PR is closed, I will fix the indentation.

**Comment by nfayazov** [comment]
@utkarshmani1997 I think it looks better now.

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**